src/chord.cpp

```diff
@@ -359,9 +359,10 @@ int Chord::CalcStem(FunctorParams *functorParams)
     this->GetYExtremes(yMax, yMin);
     params->m_chordStemLength = yMin - yMax;
 
-    int staffY = staff->GetDrawingY();
     int staffSize = staff->m_drawingStaffSize;
-    params->m_verticalCenter = staffY - params->m_doc->GetDrawingDoubleUnit(staffSize) * 2;
+
+    params->m_verticalCenter
+        = staff->GetDrawingY() - params->m_doc->GetDrawingUnit(staffSize) * (staff->m_drawingLines - 1);
```

Comment:
Instead of moving two lines down from top it really points to the vertical center of a staff (important for staves that have not 5 lines).  
Alternatively we could limit automatic stem direction to staves with five lines.